awscli/init.sql

```diff
@@ -0,0 +1,12 @@
+CREATE USER IF NOT EXISTS 'admin'@'%' IDENTIFIED BY 'Pa55WD';
+SELECT user FROM mysql.user;
+create database flask_db;
+grant ALL on flask_db.* to 'admin'@'%';
+SHOW DATABASES;
+
+
+
+export MYSQL_USER="admin"
```

Comment:
These env vars are required on APP instance, not DB one

Reply:
copied here temporary, bucket doesn't have those lines